docs/insomnia/introduction-to-plugins.md

```diff
@@ -91,23 +91,55 @@ Example: Plugin package.json file
 }
 ```
 
+### Expose Response Body Text
```

Comment:
Essentially the example below is showing how to change the response body when a request is run. The question in the discussion came because the response body works with Node JS Buffers and they wanted either:
a) an updated API to change request body
b) an example of working with buffers (current API) to change request body

The example ties into `responseHooks` and shows how to work with the Node JS Buffers by:
- converting a buffer to a string
- trying to parse the string to a JS object
- modifies the response (via a random number, and a prompt to the user)
- converts the JS object to string and then to a buffer

Let me know if you need any other context to update the description and header here! :)